### ISSUE TITLE
Add a way for the modules to be imported non-dynamically

### ISSUE THREAD

**Issue by millii19** [WFCD/warframe-items#51]
# Issue
I would like to use this library in a small mobile app using React Native (CRNA)
Unfortunately the Metro bundler (used when initiating a RN App) doesn't support dynamic imports.

# Suggestion
Make it possible to import all items statically
_Or_
Expose the JSON data files so we can import them directly

**Comment by TobiTenno** [comment]
Sorry, what exactly are you trying to do that you can't do? The standard practice for node module consumption is exposing the entrypoint for the module in the application.json.

As the sample code mentions, if you construct a new "WarframeItems" from the standard
```js
const Items = require('warframe-items')'
const items = new Items(options);
````

where options is usually a list of types you want to include. Those are all appended to an array and returned. None of the json files are downloaded at runtime nor do they require extra processing.

Those are generated only when the update is run, and that's not needed for normal usage, only when publishing on npm. Is the issue that it loads data from the JSON files when the request is made?

The problem loading all of them at creation and giving all objects all the time is that they are rather massive when loading a lot at once.

**Comment by millii19** [comment]
I'm sorry if I wasn't clear.
The following line from the index.js file is a dynamic require:
`const items = require('./data/json/${category}.json')`

I believe the reason is that the string can change during runtime.
Some bundlers don't support this (e.g. Metro Bundler)

**Comment by TobiTenno** [comment]
Right, that's kind of the point of dynamic imports.... otherwise a case for each combination and concatenation has to be written...

From a standpoint of maintaining it, it doesn't make sense to move from a dynamic setup to a static, because it will increase the necessary work to maintain it or to add another category in the future.

**Comment by TobiTenno** [comment]
And the files can be directly imported from `./node_modules/warframe-items/data/{option here}.json` if that's the route you _must_ take.

**Comment by millii19** [comment]
It actually breaks the build.

And because you mentioned maintainability: I 100% agree with you, but adding just one case for 'All' would be just as maintainable and solve this issue.

**Comment by TobiTenno** [comment]
1. It really wouldn't, because you'd be importing several megabytes of json files into memory for your application, and 
2. The problem, however, from what I've been reading up on some of their issues, is that it they scan for both, and if they can't type it, it fails.

**Comment by Kaptard** [comment]
Does it work for you to fetch the files from `/data/json` automatically pre-build and then require them locally? 

As far as I'm aware, even if we put manual conditions to load these files explicitly, bundlers like webpack or rollup won't actually recognize them. They'd have to be loaded globally, which is just terrible for memory usage as these files are quite big. Even adding just one static import for all.json would result in 10MB of memory consumption.

If you have any other idea how to avoid this, please let me know. I don't mind putting some extra complexity into this as long as it's reasonable. But adding overhead memory usage would make any sane dev ignore this package and just fetch the files manually anyways.
